src/gossip/shards.zig

```diff
@@ -4,34 +4,32 @@ const AutoHashMap = std.AutoHashMap;
 
 const Hash = @import("../core/hash.zig").Hash;
 
-const crds = @import("./crds.zig");
-const CrdsValue = crds.CrdsValue;
-const CrdsData = crds.CrdsData;
-const CrdsVersionedValue = crds.CrdsVersionedValue;
-const CrdsValueLabel = crds.CrdsValueLabel;
+const gossip = @import("./data.zig");
+const GossipDataWithSignature = gossip.GossipDataWithSignature;
+const GossipData = gossip.GossipData;
+const GossipVersionedData = gossip.GossipVersionedData;
+const GossipKey = gossip.GossipKey;
 
 const KeyPair = std.crypto.sign.Ed25519.KeyPair;
 const RwLock = std.Thread.RwLock;
 
-const CrdsPull = @import("./pull_request.zig");
+const hashToU64 = @import("./pull_request.zig").hashToU64;
 
-pub const CRDS_SHARDS_BITS: u32 = 12;
-pub const CRDS_SHARDS_LEN: u32 = 1 << CRDS_SHARDS_BITS;
+pub const GOSSIP_SHARDS_BITS: u32 = 12;
+pub const GOSSIP_SHARDS_LEN: u32 = 1 << GOSSIP_SHARDS_BITS;
 
-pub const CrdsShards = struct {
-    // shards[k] includes crds values which the first shard_bits of their hash
-    // value is equal to k. Each shard is a mapping from crds values indices to
+pub const GossipShards = struct {
```

Comment:
What do you think about "GossipTableShards"? to make it clear this is about sharding within the table specifically, and not for example about sharding data across the network or anything like that.

Reply:
agreed (7f7f55a)